### DISCUSSION TITLE
mountain duck support netapp storageGRID(s3 storage)

### DISCUSSION THREAD

**Discussion by HCjoy** [iterate-ch/cyberduck#16170] [Q&A]
Mountain duck : 版本4.16.1（22256）
S3 storage : netapp storageGRID 11.8
Client os : windows , MACos
 
When I use mountain duck mount s3 to local driver, 
1. rename file(aaaa to dddd) has error
2. rename not empty folder has error
3. rename empty folder is normal

Sofie Vogt commented:
The issue is caused by StorageGRID not supporting the necessary S3 ACLs. Currently, there is no option from our side to work around the issue.

i want to mountain duck can support netapp s3 storage(storageGRID) rename file/not empty folder

**Comment by dkocher** [comment]
You should be able to workaround the issue by setting the [hidden configuration option](https://docs.mountainduck.io/mountainduck/preferences/#hidden-configuration-options) 

```
queue.upload.acl.change=false
```

This should cause files to uploaded to S3 without setting a canned ACL in the `PUT` request or any additional `PutObjectAcl` request.

**Reply by HCjoy** [reply]
Thank you very much for your response. I have correctly configured the settings according to your method, but the situation has not improved. There might be other reasons.

mac: rename file or not empty folder
error code 36
